bindings/wasm/Cargo.toml

```diff
@@ -7,6 +7,6 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-js-sys = "0.3.64"
```

Comment:
nitpick: Any reason for that change? It essentially just means ">= 0.3.64", so it's a little stronger than "0.3", but still makes sense.

Reply:
Cloudflare worker crate is pinned to specific version which was different at patch level.

Reply:
This is fine for now cargo will pick the latest version anyways and I don't believe we actually use code that requires a higher version but if somehow we do and a user runs into that also somehow we can fix it in a non-breaking way.